### PR TITLE
feat: initialize icons package

### PR DESCRIPTION
**Pull request by gfellerph**
Added a script to download and transform icons from censhare. The script and tests are set up and icons can be downloaded using a proxy, but other things have to be implemented yet:

- [x] Writing JSON files for successful downloads, failed downloads and 404 icons
- [x] Adding a web component capable of loading and displaying the icons
- [x] #970
- [x] #971 
- [x] Add functionality to filter certain icon ranges that should be excluded from the library
- [x] Check if the <symbol> wrapper really is necessary or if the svg content can be set by using `innerHTML={svg}` and still be colorable

### CODE REVIEW

**Comment by swisspost-devs** [comment]
Preview environments are getting posted here, as soon as they are ready!

File: packages/components/src/components/post-icon/readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# post-icon
```
Comment (alizedebray) [inline packages/components/src/components/post-icon/readme.md:1]: This file should not be pushed : the component's readme.md are now part of .gitignore

File: packages/icons/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Design System Icons
+
+TODO
```
Comment (alizedebray) [inline packages/icons/README.md:3]: Are there any tickets open for the TODOs?

File: packages/icons/svgo.config.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      },
+    },
+    {
+      name: 'removeAttrs',
+      params: {
+        attrs: '(width|height|style)',
+      },
+    },
+    /* {
+      name: 'removeDimensions',
```
Comment (alizedebray) [inline packages/icons/svgo.config.ts:25]: Is this comment necessary?

File: packages/components/src/components/post-icon/readme.md (hunk 1 above)
Comment (gfellerph) [reply packages/components/src/components/post-icon/readme.md:1]: I'm not sure if we should not track the readme files. It's possible to add custom descriptions on top of the generated table and readme files are automatically displayed on github, for example here: [post-icon](https://github.com/swisspost/design-system/tree/feat-initialize-icons-package/packages/components/src/components/post-icon). WDYT?

File: .github/workflows/fetch-icons.yaml [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+on:
+  schedule:
+#            ┌───────────── minute (0 - 59)
+#            │ ┌───────────── hour (0 - 23)
+#            │ │ ┌───────────── day of the month (1 - 31)
+#            │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+#            │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+#            │ │ │ │ │
+    - cron: '0 0 * * 0'  # the job will run every Sunday at 0:00
```
Comment (oliverschuerch) [inline .github/workflows/fetch-icons.yaml:20]: very nice 👍🏼

File: packages/components/src/components/post-icon/readme.md (hunk 1 above)
Comment (oliverschuerch) [reply packages/components/src/components/post-icon/readme.md:1]: Would push it, even if it's completely auto-generated.

File: packages/documentation/package.json [hunk 5]
```diff
@@ -56,6 +57,8 @@
     "cypress": "12.5.1",
     "cypress-each": "1.13.1",
     "cypress-storybook": "0.5.1",
+    "prettier": "2.8.3",
+    "react-jsx-parser": "^1.29.0",
     "react-markdown": "8.0.5",
     "rehype-raw": "^6.1.1",
```
Comment (oliverschuerch) [inline packages/documentation/package.json:63]: remove ^

File: packages/documentation/src/stories/foundation/icons/icons.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+    <div className="w-50 py-3">
+      <p className="text-muted fs-tiny">Fade animation:</p>
+      <post-icon name="1028" animation="fade" />
+    </div>
+    <div className="w-50 py-3">
+      <p className="text-muted fs-tiny">Throb animation:</p>
+      <post-icon name="2063" animation="throb" />
+    </div>
+  </Story>
+</Canvas>
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/foundation/icons/icons.stories.mdx:275]: We should move this to the post-icon component documentation

File: packages/documentation/src/stories/foundation/icons/icons.stories.mdx [hunk 7]
```diff
@@ -0,0 +1,288 @@
...
+
+## Icons
+
+<!-- this is an invisible, navigation-only story -->
+
+<Story name="Icons">
+  <div />
+</Story>
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/foundation/icons/icons.stories.mdx:283]: Remove this, if we extract the other stories to a post-icon component documentation

File: packages/documentation/src/stories/foundation/icons/icons.stories.mdx [hunk 8]
```diff
@@ -0,0 +1,288 @@
...
+  graphics.
+</p>
+
+The icons are designed to work with the Swiss Post Design System components. They are SVGs, so they scale and can be styled with CSS.
+
+<!-- this is an invisible, navigation-only story -->
+
+<Story name="Start">
+  <div />
+</Story>
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/foundation/icons/icons.stories.mdx:29]: Remove this, if we extract the other stories to a post-icon component documentation

File: packages/icons/src/index.spec.ts [hunk 9]
```diff
@@ -0,0 +1,5 @@
+describe('index', () => {
```
Comment (oliverschuerch) [inline packages/icons/src/index.spec.ts:1]: Write tests or remove this file

File: packages/documentation/src/stories/foundation/icons/icons.stories.mdx (hunk 6 above)
Comment (gfellerph) [reply packages/documentation/src/stories/foundation/icons/icons.stories.mdx:275]: Moved this to a separate issue #1083

File: packages/documentation/src/stories/foundation/icons/icons.stories.mdx (hunk 7 above)
Comment (gfellerph) [reply packages/documentation/src/stories/foundation/icons/icons.stories.mdx:283]: Moved this to a separate issue #1083

File: packages/documentation/src/stories/foundation/icons/icons.stories.mdx (hunk 8 above)
Comment (gfellerph) [reply packages/documentation/src/stories/foundation/icons/icons.stories.mdx:29]: Moved this to a separate issue #1083

File: packages/icons/src/index.spec.ts (hunk 9 above)
Comment (gfellerph) [reply packages/icons/src/index.spec.ts:1]: Moved to a separate issue #1084

File: packages/icons/svgo.config.ts (hunk 3 above)
Comment (gfellerph) [reply packages/icons/svgo.config.ts]: Removed it

File: packages/icons/README.md (hunk 2 above)
Comment (gfellerph) [reply packages/icons/README.md]: I added a readme and a contributing.md

File: packages/components/src/components/post-icon/readme.md (hunk 1 above)
Comment (gfellerph) [reply packages/components/src/components/post-icon/readme.md:1]: Removed the line in the .gitignore, readmes are now tracked in git.

Comment (alizedebray) [reply packages/components/src/components/post-icon/readme.md:1]: Sure, we can push them too.
Could you please remove the path from .gitignore then?

**Review by alizedebray — APPROVED** [review]

File: packages/documentation/package.json (hunk 5 above)
Comment (oliverschuerch) [reply packages/documentation/package.json]: @gfellerph Did you see this CR?

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**